indico/modules/events/layout/default.py

```diff
@@ -97,9 +97,9 @@ def get_default_menu_entries():
             visible=_visibility_call_for_abstracts,
         ),
         MenuEntryData(
```

Comment:
Yes. If you add it maybe put it in the same place (and with the same name) as this one here so we get a conflict and don't end up with two functions for the same thing when merging those PRs:

https://github.com/indico/indico/pull/2658/files#diff-377bf7cb1677018ce7c94b1f7bc116d5R114